drivers/docker/stats.go

```diff
@@ -141,13 +136,22 @@ func (h *taskHandle) collectStats(ctx context.Context, destCh *usageSender, inte
 			retry++
 			continue
 		}
+		defer statsReader.Body.Close()
+
+		var stats containerapi.Stats
+		if err := binary.Read(statsReader.Body, binary.LittleEndian, &stats); err != nil {
+			h.logger.Error("error decoding stats data for container", "error", err)
+		}
```

Comment:
is there a reference for where this came from? manually reading little endian seems out of left field